### PR TITLE
Adding documentation, examples for new default grammar

### PR DESCRIPTION
**Pull request by DarrinEide**

### CODE REVIEW

File: academic-services/project-academic-knowledge/concepts-queries.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* **title**: Full title (Ti) or unique words from title (W)
+* **year**: Publication year (Y)
+
+## Stage 2: Generate valid interpretation hypotheses
+
+:::image type="content" source="media/concept-semantic-interpretation-hypothesis.png" alt-text="Semantic Interpretation Generation Architecture - Generate interpretation hypotheses":::
+
+Semantic interpretation generation is broken into two important steps. The first of these is generating interpretation hypotheses using a [Context Sensitive Grammar (CSG)](https://en.wikipedia.org/wiki/Context-sensitive_grammar) conforming to the [Speech Recognition Grammar Specification (SRGS)](https://www.w3.org/TR/speech-grammar/).
+
+Generally SRGS grammars are context-free, meaning grammar rules can be matched lacking any additional context of previously matched rules. The Interpret method changes this behavior by including a *grammar context* that enables rules to reference metadata from previously matched rules.
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/concepts-queries.md]: >Generally [](start = 0, length = 9)

nit: "Generally, "

File: academic-services/project-academic-knowledge/concepts-queries.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+It's important to understand that each hypothesis generates *sub-hypotheses* in the form of an expansion-tree, and each tree branch *gets its own unique copy of the grammar context*. This allows us to generate many potential hypothesis that fully explore the search space for a given query.
+
+## Stage 3: Resolve valid interpretation hypotheses
+
+:::image type="content" source="media/concept-semantic-interpretation-resolve.png" alt-text="Semantic Interpretation Generation Architecture - Resolve valid hypothesis":::
+
+Once a valid hypothesis has been found, the Interpret method *resolves* the hypothesis into an interpretation by generating and storing the following:
+
+* A probability score for the hypothesis. This is the static rank of the top-most entity returned by the hypothesis plus any alternative [weight penalties imposed by the SRGS grammar](https://www.w3.org/TR/speech-grammar/#S2.4)
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/concepts-queries.md]: >static rank [](start = 54, length = 11)

might want to call out that this is the "static MAG rank".

File: academic-services/project-academic-knowledge/reference-interpret-method.md [hunk 3]
```diff
@@ -40,53 +44,128 @@ Name | Description
 **interpretations[x].rules[y].name** | Name of the rule.
 **interpretations[x].rules[y].output** | Output of the rule.
 **interpretations[x].rules[y].output.type** | The data type of the output of the rule.  For the Academic Knowledge API, this will always be "query".
-**interpretations[x].rules[y].output.value** | he output of the rule. For the Academic Knowledge API, this is a query expression string that can be passed to the evaluate and calchistogram methods.
-**aborted** | True if the request timed out.
+**interpretations[x].rules[y].output.value** | The output of the rule. For the Academic Knowledge API, this is a query expression string that can be passed to the evaluate and calchistogram methods.
+**interpretations[x].rules[y].output.entities** | Optional array of top entities matching the interpretation.
+**timed_out_count** | The number of underlying index partitions that timed out when generating interpretations.
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/reference-interpret-method.md]: >The number of underlying index partitions [](start = 22, length = 41)

do we want to expose the underlying arch to users?  Not sure what they would do with this information...

File: academic-services/project-academic-knowledge/reference-interpret-method.md [hunk 4]
```diff
@@ -40,53 +44,128 @@ Name | Description
...
+## Examples
+
+### Generate academic query suggestions
 
-### Example:
+This example generates the most likely query suggestions (completions) for a partial author query "author: j tee".
 
 ``` HTTP
-https://api.labs.cognitive.microsoft.com/academic/v1.0/interpret?query=papers by jaime&complete=1&count=2
+https://api.labs.cognitive.microsoft.com/academic/v1.0/interpret?query=author:%20jaime%20t&complete=1&count=2
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/reference-interpret-method.md]: >author:%20jaime%20t [](start = 71, length = 19)

Should this be "author:%20j%20tee" to match description on line 56?

File: academic-services/project-academic-knowledge/reference-interpret-method.md (hunk 3 above)
Comment (DarrinEide) [reply academic-services/project-academic-knowledge/reference-interpret-method.md]: I was asking myself that question as I was writing this. I need to say something about it, as if I just ignore it is highly confusing what the heck it is.

Any suggestions for how I could make it more relatable?

---
In reply to: [395375353](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/130#discussion_r395375353) [](ancestors = 395375353)

File: academic-services/project-academic-knowledge/reference-interpret-method.md [hunk 5]
```diff
@@ -40,53 +44,128 @@ Name | Description
...
+
+* **complete=1**: Indicates that query completions should be generated
+* **count=2**: Indicates that only two interpretations should be generated
+
+As you can see from the response below, the most likely interpretations that complete the partial author query *author: j tee* are:
+
+* *author: jaime teevan*
+* *author: john r teerlink*
+
+The service generated query completions instead of considering only exact matches for the author *j tee*. Note that the canonical value *j l green* matched via the synonym *jamie green*, as indicated in the parse.
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/reference-interpret-method.md]: >j l green* matched via the synonym *jamie green*, [](start = 137, length = 49)

Is this relevant?  I don't see this in the JSON or above.

File: academic-services/project-academic-knowledge/reference-interpret-method.md (hunk 3 above)
Comment (RickRogahn) [reply academic-services/project-academic-knowledge/reference-interpret-method.md]: Hmm, we can say this is a parameter that is used internally to support the service.

**Pull request merged**